### PR TITLE
feature/csharp layer

### PR DESCRIPTION
**Pull request by austincrft**
Add basic csharp-layer functionality (test-runner currently commented out)

### CODE REVIEW

**Comment by austincrft** [comment]
Haha, sorry. I wasn't quite done with the README at that point. I didn't realize that my commits would automatically be added to the PR, but that's pretty nice.

Should be good to go now.

File: src/cljs/proton/layers/lang/csharp/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Add `:lang/csharp` to your layers.
+
+### Requirements
+
+- [ASP.NET 5](https://github.com/aspnet/Home)
+
+For installation of ASP.NET 5, you can view the guides on [docs.asp.net](https://docs.asp.net)
+
+* [Guide: Installing ASP.NET 5 on Linux](http://docs.asp.net/en/latest/getting-started/installing-on-mac.html)
+* [Guide: Installing ASP.NET 5 on Mac OS X](http://docs.asp.net/en/latest/getting-started/installing-on-linux.html)
```
Comment (dvcrn) [inline src/cljs/proton/layers/lang/csharp/README.md]: you mixed the mac and linux links up

**Comment by austincrft** [comment]
I certainly did. Fixed.

**Comment by dvcrn** [comment]
Cool! Congrats on your first layer! :smile:

**Pull request merged**